src/js/worklets/onboarding_carousel.js

```diff
@@ -1,6 +1,8 @@
 import { useDerivedValue, withTiming, Easing } from 'react-native-reanimated';
 
 const slideAnimationDuration = 300;
+const pageSize = 25;
```

Comment:
If page size is `100/totalPages` then imo it would be better explicitly put that in the code.

@rahulpsingh wdyt?